Automattic/jetpack
# Search: use classic search for ios<16 devices

Fixes #33495 

## Proposed changes:

Enable Classic Search if browser is iOS and version < 16. **A follow up task will be needed to enable the compatibility patch on WPCOM.**

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Prepare local / JN site with Instant Search enabled
* Open `/?s=test`
* Ensure search overlay pops up
* Choose a device with iOS < 16 on browserstack <- look at the secret store for account
* Refresh the page
* Ensure classic search returned results too
* Ensure all tests pass

<img width="634" alt="image" src="https://github.com/Automattic/jetpack/assets/1425433/fec8b667-936e-4007-b168-f284a196e6d6">
